noxfile.py

```diff
@@ -127,11 +133,11 @@ def get_sphinx_build_command(kind):
         return [
             "sphinx-build",
             "-W",
-            "-c", "docs/html",  # see note above
-            "-d", "docs/build/doctrees/" + kind,
-            "-b", kind,
-            "docs/" + kind,
-            "docs/build/" + kind,
+            "-c=docs/html",  # see note above
+            f"-d=docs/build/doctrees/{kind}",
```

Comment:
Is it me or this feels like a hack?

Reply:
Using f-strings feels like a hack? I was going to do this anyway, and black forcing my move to be sooner is OK with me. :shrug:

Reply:
not f-strings, putting two pieces into one item

Reply:
Agreed, I feel that we shouldn't try to fight the tool like this. It's up to black to have a way to allow us to express our intent here, which is to have two arguments, "-c" and "docs/html" which are separate but related. That's the semantics of the command line we'd normally write in the shell, and we should be able to reflect that readably in code.

IMO we just add `#fmt: off` to signal that black doesn't do what we want (see [here](https://black.readthedocs.io/en/stable/the_black_code_style.html?highlight=%23fmt#code-style))

Reply:
Yes, this is exactly what I meant